### DISCUSSION TITLE
Retrieving all Assets from a folder in the Content Builder

### DISCUSSION THREAD

**Discussion by valoote** [Accenture/sfmc-devtools#1490] [Q&A] [answered]
Hello folks,

First I want to thank Jörn and Doug as this tool is amazing, especially when dealing with multiple BUs.

My request for today:

I have a folder at the root of the Content Builder containing a bunch of assets that I'm using for an email template.

Is there a way to filter the retrieve command to only download assets from that folder (and its subfolders if any)?

Thank you!

**Comment by JoernBerkefeld** [comment, marked as answer]
hi @valoote thank you for your patience!
Yes, there is an option: the `--like` option, to be exact. Try running something like this:

The following will empty your `retrieve/cred/bu/asset` folder and then ONLY save assets that sit directly in my/folder or a subfolder of my/folder, e.g. my/folder/otherFolder
```
mcdev r cred/bu asset --like.r__folder_Path "Content Builder/my/folder%"
```

The following will empty your `retrieve/cred/bu/asset` folder and then ONLY save assets that sit in a subfolder of my/folder, e.g. my/folder/otherFolder. Note the trailing forward-slash which makes sure nothing directly in my/folder will be saved to your disk
```
mcdev r cred/bu asset --like.r__folder_Path "Content Builder/my/folder/%"
```

Note that it still downloads everything and THEN decides what to actually save to disk.
